packages/nx/src/utils/package-manager.ts

```diff
@@ -88,6 +88,9 @@ export function getPackageManagerCommand(
       };
     },
     npm: () => {
+      // TODO: Remove this
```

Comment:
is this just temporary or is this supposed to be removed in future versions of nx?